### PR TITLE
CI build for ndcube 1.3 branch

### LINKED ISSUE #151: Set up testing against the ndcube 1.3 branch as well as ndcube master
Hi @nabobalis, the recent API breaking changes to the ndcube master branch mean that testing ```sunraster``` against does not reflect whether ```sunraster``` will work with the stable release of ```ndcube```.  Is it possible to set up additional CI tests that use the ndcube 1.3 branch, so we can confirm that ```sunraster``` v0.1 will work with ```ndcube``` v1.3.x?  Is that a difficult job?

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #151

I don't think this is worth merging in.

### CODE REVIEW

**Comment by nabobalis** [comment]
Now I am getting can't import `sunraster.raster`

**Comment by DanRyanIrish** [comment]
Thanks @nabobalis for this PR.  Why do you say it's not worth merging?

File: docs/data_classes.rst [hunk 1]
```diff
@@ -109,70 +109,70 @@ Coordinates
...
 `~astropy.units.Quantity` objects because longitude and latitude are couple dimensions.
 These properties inspect the WCS and extra coords objects and locate where and
 how the relevant coordinate information is stored.
 This is possible only if the coordinate name is supported by `sunraster`.
-This see these supported names, see ``sunraster.raster.SUPPORTED_LONGITUDE_NAMES``,
-``sunraster.raster.SUPPORTED_LATITUDE_NAMES``,
-``sunraster.raster.SUPPORTED_SPECTRAL_NAMES``, and
-``sunraster.raster.SUPPORTED_TIME_NAMES``.
+This see these supported names, see ``sunraster.SpectrogramCube.SUPPORTED_LONGITUDE_NAMES``,
+``sunraster.SpectrogramCube.SUPPORTED_LATITUDE_NAMES``,
```
Comment (DanRyanIrish) [inline docs/data_classes.rst:159]:
```suggestion
``sunraster.spectrogram.SUPPORTED_LATITUDE_NAMES``,
```

File: docs/data_classes.rst [hunk 2]
```diff
@@ -109,70 +109,70 @@ Coordinates
...
+``sunraster.SpectrogramCube.SUPPORTED_SPECTRAL_NAMES``, and
```
Comment (DanRyanIrish) [inline docs/data_classes.rst:160]:
```suggestion
``sunraster.spectrogram.SUPPORTED_SPECTRAL_NAMES``, and
```

File: docs/data_classes.rst [hunk 3]
```diff
@@ -109,70 +109,70 @@ Coordinates
...
+``sunraster.SpectrogramCube.SUPPORTED_TIME_NAMES``.
```
Comment (DanRyanIrish) [inline docs/data_classes.rst:161]:
```suggestion
``sunraster.spectrogram.SUPPORTED_TIME_NAMES``.
```

File: docs/data_classes.rst [hunk 4]
```diff
@@ -109,70 +109,70 @@ Coordinates
...
 If the coordinate name cannot be found, these properties raise an error.
 
-In addition to the four primary coordinates, `~sunraster.Raster` also provides a
-convenience for the exposure time, `sunraster.Raster.exposure_time`.
+In addition to the four primary coordinates, `~sunraster.SpectrogramCube` also provides a
+convenience for the exposure time, `sunraster.SpectrogramCube.exposure_time`.
 The supported exposure time coordinate names can be found under
-``sunraster.raster.SUPPORTED_EXPOSURE_NAMES``.
+``sunraster.SpectrogramCube.SUPPORTED_EXPOSURE_NAMES``.
```
Comment (DanRyanIrish) [inline docs/data_classes.rst:167]:
```suggestion
``sunraster.spectrogram.SUPPORTED_EXPOSURE_NAMES``.
```

File: sunraster/spectrogram.py [hunk 5]
```diff
@@ -2,9 +2,10 @@
 import textwrap
 
 import numpy as np
+from ndcube.ndcube import NDCube
+from ndcube.utils.cube import convert_extra_coords_dict_to_input_format
+
```
Comment (DanRyanIrish) [inline sunraster/spectrogram.py:7]:
```suggestion
```

File: sunraster/spectrogram_sequence.py [hunk 6]
```diff
@@ -1,13 +1,13 @@
-import textwrap
 import numbers
+import textwrap
 
 import numpy as np
-import astropy.units as u
-import ndcube.utils.sequence
 from ndcube import NDCubeSequence
```
Comment (DanRyanIrish) [inline sunraster/spectrogram_sequence.py:6]:
```suggestion
```

**Comment by DanRyanIrish** [comment]
@nabobalis, as part of this PR, can we set up code coverage reports for the sunraster CI?

File: sunraster/spectrogram_sequence.py (hunk 6 above)
Comment (nabobalis) [reply sunraster/spectrogram_sequence.py:6]: If you want this, the isort config needs changing.

**Comment by nabobalis** [comment]
I added the codecov file. Hopefully it works.

**Comment by nabobalis** [comment]
You can see it here: https://codecov.io/gh/sunpy/sunraster I forget if master needs to run first before it reports numbers.

File: sunraster/spectrogram_sequence.py (hunk 6 above)
Comment (DanRyanIrish) [reply sunraster/spectrogram_sequence.py:6]: Ah!  Is that what isort is to do with!  I had no idea why it was failing.  Where can I find the codestyle rules that will make it pass?

Comment (nabobalis) [reply sunraster/spectrogram_sequence.py:6]: I think they are in the setup.cfg file

Comment (kakirastern) [reply sunraster/spectrogram_sequence.py:6]: @DanRyanIrish You can probably use https://github.com/asottile/seed-isort-config to help you to populate the `known_third_part` list to pass the `isort` test locally. Then commit all the changes first in the local environment before pushing the changes to the repo. You don't want `isort` to make changes "remotely" for you.

Comment (nabobalis) [reply sunraster/spectrogram_sequence.py:6]: I used isort locally. Isort is not run by the CI.

**Pull request merged**